trackers/core/reid/callbacks.py

```diff
@@ -0,0 +1,77 @@
+from typing import Any, Optional
+
+import wandb
+from torch.utils.tensorboard import SummaryWriter
+
+
+class BaseCallback:
+    def on_train_batch_start(self, logs: dict, idx: int):
+        pass
+
+    def on_train_batch_end(self, logs: dict, idx: int):
+        pass
+
+    def on_validation_batch_start(self, logs: dict, idx: int):
+        pass
+
+    def on_validation_batch_end(self, logs: dict, idx: int):
+        pass
+
+    def on_checkpoint_save(self, checkpoint_path: str, epoch: int):
+        pass
+
+    def on_end(self):
+        pass
+
+
+class TensorboardCallback(BaseCallback):
```

Comment:
I think it would be cool to have one more callback that would save `matplotlib` chart as well.